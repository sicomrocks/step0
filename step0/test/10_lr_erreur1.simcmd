#Test fonction lr ; mauvais registre d'entrée

lr HI 0x12a
lr invalid_reg 0x12a #cette commande génère une erreur
lr $12 0xd
