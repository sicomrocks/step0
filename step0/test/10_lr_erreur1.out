HI : 0x0000012a
